agent/proxycfg/manager_test.go

```diff
@@ -447,7 +446,7 @@ func assertWatchChanRecvs(t *testing.T, ch <-chan *ConfigSnapshot, expect *Confi
 
 func TestManager_deliverLatest(t *testing.T) {
 	// None of these need to do anything to test this method just be valid
-	logger := log.New(os.Stderr, "", log.LstdFlags)
+	logger := testutil.TestHcLog(t.Name())
```

Comment:
This should ultimately funnel logs through a `testutil.TestWriter` and helpfully you already have a `t` in scope so it's not a difficult fix.